docs/source-app/workflows/byoc/index.rst

```diff
@@ -5,62 +5,287 @@ Run Apps on your own cloud (BYOC)
+personal credentials (this could include your CLI profile name, secret, and key) for delegated
+access and an identity provider for secure access to the data plane.
+
+After submitting a cluster creation request, the Lightning Control Plane creates the required cloud
+infrastructure on the user cloud account. On AWS this includes creating the EKS kubernetes cluster,
+necessary S3 buckets, IAM roles, and Lighting agent managing this environment.
+
+----
+
+***********************************
+What does Lightning BYOC do for me?
+***********************************
+
+Lightning BYOC allows you to use Lightning while keeping your data behind a wall.
+Lightning can interact with your data, but your data never leaves your cloud.
+This means you can run Apps, train Apps, collaborate on creating Apps and component
+within your own cloud. So your data is protected and you save money.
+
+----
+
+*************************
+What are you going to do?
+*************************
+
+The goal of this document is to get you to run a Lightning App on your cloud/cluster.
+There are other things you can do with Lightning BYOC, but running an App on your own
+cloud/cluster will show you that the Lightning cluster works.
+
+At a high level, to accomplish that, you need to do these things:
+
+#. Give Lightning permission to act on your behalf in your AWS account.
+#. Install Lightning.
+#. Create a Lightning cluster and then view the list of clusters and cluster logs.
+#. Run an App on your cluster.
+
+----
+
+*************
+Prerequisites
+*************
+
+For Lightning and Lightning BYOC:
+
+* Python 3.8.x or later (3.8.x, 3.9.x, 3.10.x) for Lightning installation
+
+* An active AWS account with enough permissions to create AWS objects
+
+(Optional) For Terraform only:
+
+* Terraform installed
+
+* AWS CLI installed
+
+* AWS credentials to sign in to AWS
+
+----
+
+******************
+Step 1: Set up AWS
+******************
+
+In this step, you are going to create a role with permissions that allow Lightning to make changes to your AWS resources on your behalf.
+You can do this directly in AWS or you can use Terraform. They both accomplish the same goal,
+so choose the method that works best for you.
+
+----
+
+Who should set up AWS?
+^^^^^^^^^^^^^^^^^^^^^^
+
+In companies that have an IT department or a department that oversees your cloud provider resources,
+they will be the ones to set up and assign/add this role to your AWS account.
+If you’re a one person army (you’re a researcher, data scientist, Lightning App creator, or more),
+you’re going to have to set up the role for your own account by yourself.
+
+.. note::
+
+    If you are an IT person setting this role up for someone else, assign the role to their account once the role is created.
+
+----
+
+Method 1: Use the AWS Console to set up AWS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You're going to create a custom role in AWS using the AWS management console.
```

Comment:
@edenlightning 

There's a short video in this file that shows them, instead of writing all the steps out. It's only visible if you view the raw file:

https://pl-flash-data.s3.amazonaws.com/assets_lightning/docs/images/storage/byoc_aws_setup_shorter.mp4